resources/lib/gui.py

```diff
@@ -1129,7 +1131,7 @@ def onClick(self, controlID):
             # Display list items
             self._display_listitems( focus = itemIndex )
         
-        elif controlID == 301:
+        elif controlID in [301, 1301]:
```

Comment:
Yeah I think it does. I will take some time for that next week if you don't mind. Currently busy replying to all (bug) reports on the forums after my big code rewrite of skinhelper ;-)